example-deployment/k8s/jobs.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: reporting-server-migrations-job
+spec:
+  # This is an Alpha feature to remove completed jobs automatically.
```

Comment:
Is there any alternative to using this alpha feature?

Reply:
The most common solution is to create a crontab job that cleans up the completed jobs every X hours.

Off topic: We also need to create a crontab job that cleans up the database table if logs are > 7 days.

Reply:
Removed alpha feature.  I didn't find an elegant way to kill a finished task if this task is not found within a cronjob. Since migration does not fall into the category of a cronjob (because it should be run on every deployment and not every certain amount of time), we should run this task as a deploy script. And we should kill that task manually.

Reply:
Removed alpha feature. I didn't find an elegant way to kill a finished task if this task is not found within a cronjob. Since migration does not fall into the category of a cronjob (because it should be run on every deployment and not every certain amount of time), we should run this task as a deploy script. And we should kill that task manually.